### DISCUSSION TITLE
Help us name the 1.6 release!

### DISCUSSION THREAD

**Discussion by tacastillo** [dagster-io/dagster#18928] [Community]
Hey folks! Tim from the Dagster team here.

We'll be releasing Dagster v1.6 soon and want your feedback!

Every major and minor release gets named after a hit single or an album from a pop diva. Sometimes, the name of the song/album is inspired by what changes are coming with the release.

For inspiration, 1.6 will have:
- a revamp of our asset graph UI
- the Dark Mode UI
- integration with AWS Lambdas through Dagster Pipes
- general improvements to partitions and sensors, such as better access to their logs

And here are some songs/albums we've previously named our versions after!

[How Will I Know?](https://www.youtube.com/watch?v=m3-hY-hlhBg) (Whitney Houston)
[Material Girl](https://www.youtube.com/watch?v=xeddEMuM0dA) (Madonna)
[Smooth Operator](https://www.youtube.com/watch?v=4TYv2PhG89A) (Sade)
[Formation](https://www.youtube.com/watch?v=WDZJPJV__bQ) (Beyonce)
[Thank U, Next](https://www.youtube.com/watch?v=gl1aHhXnN1k) (Ariana Grande)
[Hello](https://www.youtube.com/watch?v=YQHsXMglC9A) (Adele)
[Cool for the Summer](https://www.youtube.com/watch?v=il9nqWw9W3Y) (Demi Lovato)
[Lucky Star](https://www.youtube.com/watch?v=ThHz9wlBeLU) (Madonna)
[The Edge of Glory](https://www.youtube.com/watch?v=QeWBS0JBNzQ) (Lady Gaga)
[Laundry Service](https://www.youtube.com/watch?v=7BPE5sR6oUU&list=PL035E8452800ACEA5) (Shakira)
[In the Zone](https://www.youtube.com/watch?v=we8h_2RFAyA) (Britney Spears)
[Waiting to Exhale](https://www.youtube.com/watch?v=_FT7ciAPuJk&list=PL_sN_rrP4od0gZ4C3LGkgcaRpVa_koQX4) (Whitney Houston, et al.)

[Listen to the full playlist](https://open.spotify.com/playlist/476UFttwzU8476ki1Aqcdd)

Drop a comment with your suggestions, and upvote the suggestions you like!

**Comment by PedramNavid** [comment]
"Back to Black" - Amy Winehouse

**Reply by wbashir** [reply]
Smooth Operator

**Comment by danielgafni** [comment]
AC/DC - "Back in Black"

**Reply by frasermarlow** [reply]
I was going with "Dark side of the Moon", but both work.

**Comment by danielgafni** [comment]
My Chemical Romance - "Welcome to the Black Parade"

**Comment by danielgafni** [comment]
David Bowie - "Blackstar" (rhymes with Dagster!)

**Comment by EmilSundman** [comment]
"Dark Horse" - Katy Perry

**Comment by bkktimber** [comment]
_Getting Older_ by Billie Eilish because when your new look is darker, and you're better at reading logs (ie. old stuff), you're definitely getting older.

**Comment by jomeier** [comment]
"Frozen" - Madonna

**Comment by matboett** [comment]
"Blackout" - Logistics

**Reply by frasermarlow** [reply]
Dating myself, but "Blackout" was an album by the Scorpions.

**Comment by tacastillo** [comment]
Looks like the winner is `"Back to Black" - Amy Winehouse`!
